src/main/GA4GHAlignment.js

```diff
@@ -118,5 +128,8 @@ class GA4GHAlignment /* implements Alignment */ {
     return alignment.fragmentName + ':' + alignment.readNumber;
   }
 }
+function makeCigarString(cigarOps: Array<CigarOp>) {
```

Comment:
It looks like this function is copied from [SamRead.js](https://github.com/hammerlab/pileup.js/blob/master/src/main/data/SamRead.js#L253). Can you move this function to Alignment.js from both SamRead and GA4GHAlignment and reference the function from Alignment.js?

Reply:
The function is actually not directly copied from SamRead.js. The function signatures are different in GA4GHAlignment.js and SamRead.js

Reply:
It would be great if we could actually change the SamRead function to match yours, considering it enforces types. We can still create a function in Alignment.js and modify the usage in SamRead.js to call the function in Alignment.js

Reply:
I made the edits, I think it should work.